rabbitmq/check.py

```diff
@@ -320,13 +320,22 @@ def get_connections_stat(self, instance, base_url, object_type, vhosts, custom_t
                               ssl_verify=ssl_verify, proxies=instance_proxy)
 
         stats = {vhost: 0 for vhost in vhosts}
+        connection_states = {}
```

Comment:
this could be a `defaultdict(int)` from `collections`, it will remove the line 325 and 326.